gdk/commands/component/PublishCommand.py

```diff
@@ -67,6 +101,32 @@ def upload_artifacts_s3(self, component_name, component_version):
         try:
             bucket = self.project_config["bucket"]
             region = self.project_config["region"]
+            encrypt = self.project_config.get("encrypt")
+            metadata = self.project_config.get("metadata")
+
+            # Encryption artifact setup
+            if (type(encrypt) is str) and (encrypt.lower() == 'true'):
+                logging.info("Using default encryption.")
+                extra_args = {'ServerSideEncryption': 'AES256'}
+            elif (type(encrypt) is str) and (encrypt.lower() == 'default'):
+                logging.info("Using default bucket encryption.")
+                kms_key_id = self.get_bucket_encryption(bucket_name=bucket)
+                extra_args = kms_key_id
+            elif (type(encrypt) is dict) and encrypt.get('kms_key_id'):
+                logging.info("Using KMS encryption.")
+                extra_args = {
+                    "ServerSideEncryption": encrypt.get('server_side_encryption'),
+                    "SSEKMSKeyId": encrypt.get('kms_key_id')
+                }
+            else:
+                logging.info("No encryption configuration found.")
+                extra_args = {}
```

Comment:
Move it all the way to the top after L106

Reply:
Fixed in new commit.